src/BrainCloudTournament.cpp

```diff
@@ -42,6 +57,18 @@ namespace BrainCloud
 		m_client->sendRequest(sc);
 	}
 
+	void BrainCloudTournament::joinDivision(const char* in_divSetId, const char* in_tournamentCode, int64_t in_initialScore, IServerCallback * in_callback)
```

Comment:
Actually, lets change all `const char*` to `const std::string&` for the new API calls.
I discussed this with @JonathanMcCaffrey previously.

Reply:
_and dont change existing ones_